scripts/tools/check_includes_config.py

```diff
@@ -117,6 +117,7 @@
 
     # Only uses <chrono> for zero-cost types.
     'src/system/SystemClock.h': {'chrono'},
+    'src/system/DurationTimer.h': {'string'},
```

Comment:
This one is used in things that are used in embedded ... so you cannot use STL that uses HEAP. this is not allowed.

Overall this duration timer seems not something we want in the SDK since we have MATTER_TRACE_SCOPE that is for scoped tracing. You can create a "trace to log" and include some filters if you want to only trace specific things.